### PR TITLE
fixed "Add via Transport" short keys

### PR DESCRIPTION
**Pull request by thorsten-wolf-neptune**
due to the lv_key being "string" suffix key component values shorter than the full length lead to a CX_SY_RANGE_OUT_OF_BOUNDS exception:
image
image

### CODE REVIEW

**Comment by larshp** [comment]
unit tests: looks like a bug in the transpiler, I'll have a look

File: src/ui/zcl_abapgit_gui_page_data.clas.abap [hunk 1]
```diff
@@ -1,3 +1,8 @@
+*----------------------------------------------------------------------*
```
Comment (mbtools) [inline src/ui/zcl_abapgit_gui_page_data.clas.abap:1]: please remove the comments added by pretty print (we don't have these anywhere else in abapGit). thanks

File: src/ui/zcl_abapgit_gui_page_data.clas.abap [hunk 2]
```diff
@@ -125,7 +130,7 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_DATA IMPLEMENTATION.
       mi_config->add_config( ls_config ).
     ENDLOOP.
 
-  ENDMETHOD.
+  ENDMETHOD.                    "add_via_transport
```
Comment (mbtools) [inline src/ui/zcl_abapgit_gui_page_data.clas.abap:133]: remove these comments

**Review by mbtools — CHANGES_REQUESTED** [review]
just some clean-up. thanks!

File: src/ui/zcl_abapgit_gui_page_data.clas.abap (hunk 1 above)
Comment (thorsten-wolf-neptune) [reply src/ui/zcl_abapgit_gui_page_data.clas.abap]: ok sorry. wasn't aware of this. Removed them now :-)

File: src/ui/zcl_abapgit_gui_page_data.clas.abap (hunk 2 above)
Comment (thorsten-wolf-neptune) [reply src/ui/zcl_abapgit_gui_page_data.clas.abap]: removed also those

**Review by larshp — APPROVED** [review]
looks good to me

**Review by mbtools — COMMENTED** [review]
👍

**Pull request merged**